### ISSUE TITLE
Type of `v16.datatypes.SampledValue.context` is incorrect

### ISSUE THREAD

**Issue by OrangeTux** [mobilityhouse/ocpp#421]
The specification allows the field to be optional. But the library requires it.

https://github.com/mobilityhouse/ocpp/blob/0adf9458b15b7acc605d3a13355f76d14f22fdc9/ocpp/v16/datatypes.py#L107

**Comment by Jared-Newell-Mobility** [comment]
Good catch - 7.43 - Optional. Type of detail value: start, end or sample. Default = “Sample.Periodic”

**Comment by Jared-Newell-Mobility** [comment]
Satisfied with PR - https://github.com/mobilityhouse/ocpp/pull/495 - For inclusion in next major release
